nullaway/src/main/java/com/uber/nullaway/NullAway.java

```diff
@@ -2279,7 +2279,7 @@ private boolean mayBeNullExpr(VisitorState state, ExpressionTree expr) {
               "whoops, better handle " + expr.getKind() + " " + state.getSourceForNode(expr));
         }
     }
-    exprMayBeNull = handler.onOverrideMayBeNullExpr(this, expr, state, exprMayBeNull);
+    exprMayBeNull = handler.onOverrideMayBeNullExpr(this, expr, exprSymbol, state, exprMayBeNull);
```

Comment:
do we worry about `exprSymbol` being null here? should the parameter be marked nullable?
should we do `Preconditions.checkNotNull` here because we know it cant be null?

taking a step back:
should we add a library model for `ASTHelpers` ?